.github/workflows/ambari.yml

```diff
@@ -33,7 +33,11 @@ jobs:
         java-version: 8
         distribution: temurin
         cache: maven
-    - name: Set up Python 2
-      run: sudo apt-get update && sudo apt-get install -y python2-dev python-is-python2
+    - name: Set up Python 3
```

Comment:
these packages should already be available in the base image, can you kindly check and exclude it if not required

Reply:
Excluded.